third_party/java/java_tools/BUILD

```diff
@@ -27,3 +27,16 @@ filegroup(
         "BUILD.pkg",
     ],
 )
+
+genrule(
+    name = "java_tools_zip",
+    srcs = [
+        ":java_tools",
+        "//third_party/java/jdk/langtools:java_compiler_jar",
+        "//third_party/java/jdk/langtools:javac_jar",
+        "//third_party/java/jdk/langtools:jdk_compiler_jar",
+    ],
+    outs = ["java_tools.zip"],
+    cmd = "zip -j $(OUTS) $(SRCS)",
```

Comment:
Sort the `$(SRCS)` for determinism, and consider adding `-D` (= do not add directory entries) and `-q` (= quiet operation).

Reply:
I am not familiar with how zip works, but nothing in the documentation suggests that sorting the inputs guarantee determinism.  `-D` doesn't help me that much. There are no directories underneath and if there were, perhaps I would want those entries archived. I added `-q`.